examples/masked_feature_extraction.py

```diff
@@ -0,0 +1,53 @@
+"""
+Example: Extract patch features on an image using a tissue mask.
+"""
+
+import os
+from glob import glob
+from PIL import Image
+import numpy as np
+from tqdm import tqdm
+
+from histocartography.preprocessing.feature_extraction import MaskedGridDeepFeatureExtractor
```

Comment:
You can import this module as: `from histocartography.preprocessing import MaskedGridDeepFeatureExtractor` instead